### PR TITLE
historic=monument compare function

### PR DESCRIPTION
**Pull request by srividyacb**
Added `historic=monument` compare function to check modification and deletion for monuments in OSM.

@geohacker @amishas157 would love your suggestions 📖

### CODE REVIEW

File: comparators/modifiedMonument.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+'use strict';
```
Comment (batpad) [inline comparators/modifiedMonument.js]: @srividyacb - you seem to have added both comparators/modifiedMonument.js and comparators/modified-monument.js - can you remove `comparaots/modifiedMonument.js` ?

Comment (srividyacb) [inline comparators/modifiedMonument.js]: ah thank you @batpad yeah deleted that file from this branch.

**Comment by geohacker** [comment]
@srividyacb this is great! Thank you! Looks like circle is failing the test because of indentation:

```
/home/ubuntu/osm-compare/comparators/modifiedMonument.js
   6:2  error  Expected indentation of 2 space characters but found 0  indent
   7:2  error  Expected indentation of 2 space characters but found 0  indent
   8:3  error  Expected indentation of 2 space characters but found 0  indent
  11:2  error  Expected indentation of 2 space characters but found 0  indent
  12:3  error  Expected indentation of 2 space characters but found 0  indent
  15:2  error  Newline required at end of file but not found           eol-last
```

Could you fix those? :)

**Comment by srividyacb** [comment]
@geohacker @batpad @amishas157 fixed all linting errors and branch conflicts. 

Would like final 👀  on this.

**Comment by amishas157** [comment]
Thanks @srividyacb . This looks 👌  to merge.

**Pull request merged**